src/main/java/eggventory/loans/Loan.java

```diff
@@ -0,0 +1,89 @@
+package eggventory.loans;
+
+import java.util.Calendar;
+
+//@@author cyanoei
+public class Loan {
+
+    private String stockCode;
+    private String matricNo;
+    private int quantity;
+    private Calendar loanDate;
+    private Calendar returnDate;
+
+    /**
+     * Temporary shorter constructor used to test the non-date attributes.
+     * @param stockCode the stockCode of the stock being loaned.
+     * @param matricNo the matric number of the person making the loan.
+     * @param quantity the quantity being loaned out.
+     */
+    public Loan(String stockCode, String matricNo, int quantity) {
+        //Parser should have determined beforehand that the Person and Stock being referred to are existing entries.
```

Comment:
nice

Reply:
actually not nice, oops, that isn't parser's job.